daemon/Dockerfile

```diff
@@ -7,8 +7,11 @@
 FROM ceph/base
 MAINTAINER Sébastien Han "seb@redhat.com"
 
+# patch is only needed until PR #7351 is merged in ceph/ceph (and backported
+# to the version used in ceph/ceph-docker)
 RUN apt-get update && apt-get -y install runit && \
-apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+    apt-get install patch && \
```

Comment:
It works as-is, but....you're probably right.  Committed a fix; it's in the
pull request now.



--
Mike Shuey

On Wed, Feb 10, 2016 at 9:22 AM, Leseb <notifications@github.com> wrote:

> In daemon/Dockerfile
> <https://github.com/ceph/ceph-docker/pull/203#discussion_r52462522>:
>
> >  RUN apt-get update && apt-get -y install runit && \
> > -apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
> > +    apt-get install patch && \
>
> just to be sure, shouldn't we use -y in apt?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph-docker/pull/203/files#r52462522>.
>